ociinstaller/db.go

```diff
@@ -12,7 +13,11 @@ import (
 // InstallDB :: Install Postgres files fom OCI image
 func InstallDB(ctx context.Context, dblocation string) (string, error) {
 	tempDir := NewTempDir(dblocation)
-	defer tempDir.Delete()
+	defer func() {
+		if err := tempDir.Delete(); err != nil {
```

Comment:
it does not work well to output text to the console during this phase, as we have progress spinners which don't really play well with console messages: 
<img width="1742" alt="image" src="https://user-images.githubusercontent.com/2838960/167837218-509a9f7b-d160-4d65-b5c1-b5144649c64f.png">

Ultimately the solution to getting these kinds of warnings out would be for Steampipe to write a log-file. For now a decent compromise would be to include this as trace logging: 

```
log.Printf("[TRACE] Failed to delete temp dir '%s' after installing db files: %s", tempDir, err)
```

Reply:
updated